torch/testing/_internal/common_methods_invocations.py

```diff
@@ -15907,6 +15910,8 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                     skips=(
                         DecorateInfo(unittest.expectedFailure, 'TestNormalizeOperators', 'test_normalize_operator_exhaustive'),
                         DecorateInfo(unittest.expectedFailure, 'TestJit', 'test_variant_consistency_jit',),
+                        # NYI: The operator 'aten::rsub.Tensor' is not currently implemented for the MPS device
+                        DecorateInfo(unittest.expectedFailure, 'TestConsistency', 'test_output_grad_match'),
```

Comment:
How are these MPS skips related to this PR?

Reply:
These were related when trying to fix a suspicious code like
https://github.com/pytorch/pytorch/blob/9dfeec9cdc246a6a003dff4b6ba0a5ceb60613f1/test/test_mps.py#L195
(expected `key = op.name + '.' + op.variant_test_name if op.variant_test_name else op.name`) but that triggered too many failures in test_mps.py that fixes are OT to this PR. I can undo the MPS related skips but I believe these represent real issues, although, OT here.

Reply:
I agree it's a weird way to format the key, but I don't think it's a problem and all the skips that you added are already listed in `test_mps.py` so I don't think we need to repeat them in the `OpInfo`.

Reply:
Not sure why the skips listed in test_mps.py were not effective at the time of testing... but I have now removed these from opinfos.

Reply:
After removing the skips, there are several MPS device related failures: https://hud.pytorch.org/pr/pytorch/pytorch/123375#23761295650